bundles/CoreBundle/Resources/migrations/Version20191015131700.php

```diff
@@ -0,0 +1,35 @@
+<?php
```

Comment:
maybe changes in install.sql are missing?

Reply:
done :)